Source/Core/VideoCommon/VideoCommon.vcxproj

```diff
@@ -171,4 +171,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Your comments are correct but this happens all the time in dolphin... Just ignore the extraneous back and forth changes🙂